File: src/main/java/de/maxhenkel/pipez/ModItemGroups.java
package de.maxhenkel.pipez;

import de.maxhenkel.pipez.blocks.ModBlocks;
import net.minecraft.world.item.CreativeModeTab;
import net.minecraft.world.item.ItemStack;

public class ModItemGroups {

    public static final CreativeModeTab TAB_PIPEZ = new CreativeModeTab("pipez") {

        @Override
        public ItemStack makeIcon() {
            return new ItemStack(ModBlocks.ITEM_PIPE.get());
        }

    };

}

File: src/main/java/de/maxhenkel/pipez/ModItemGroups.kt
package de.maxhenkel.pipez

import de.maxhenkel.pipez.blocks.ModBlocks
import net.minecraft.world.item.CreativeModeTab
import net.minecraft.world.item.ItemStack

object ModItemGroups {
    val TAB_PIPEZ = object : CreativeModeTab(Main.MODID) {
        override fun makeIcon(): ItemStack {
            return ItemStack(ModBlocks.ITEM_PIPE)
        }
    }
}